docker/kind/metrics-server.yaml

```diff
@@ -0,0 +1,196 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    k8s-app: metrics-server
+  name: metrics-server
+  namespace: kube-system
```

Comment:
Just curious why this is living in the kube-system namespace as opposed to say, default?

Reply:
excellent question

Reply:
:) I honestly am not sure - I think just because it's in the same category of cluster wide functionality etc on kube. 

This is the actual server that calculates usage statistics across the cluster - and this yaml was ripped directly from their deploy with very minor tweeks.